identity_providers_test.go

```diff
@@ -0,0 +1,77 @@
+package keycloak
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	kcHost     = "localhost:8080"
```

Comment:
I guess you need to have a Keycloak instance running locally... We don't have unit tests yet for this calls but if you wan't to write some, you should check in the bridge how you can dynamically instanciate a HTTP server with mocks.